mage_integrations/mage_integrations/sources/stripe/__init__.py

```diff
@@ -368,10 +366,11 @@ def get_discovery_metadata(schema, key_properties, replication_method, replicati
     return metadata.to_list(mdata)
 
 
-def discover() -> Dict:
+def discover(logger=LOGGER):
```

Comment:
Don't set an mutable object as the default value.
Do
```
df discover(logger=None):
    if logger is None:
        logger = LOGGER
```

Reply:
Modified on 028336556fca308242837d69d26186c17481362d